tests/Nox.Types.Tests/EntityFrameworkTests/Models/StreetAddress.cs

```diff
@@ -0,0 +1,23 @@
+namespace Nox.Types.Tests.EntityFrameworkTests;
+
+public class StreetAddressId : ValueObject<int, StreetAddressId>
+{ }
+
+public sealed class StreetAddress
```

Comment:
@AndreyDegtyarev  we can speak tomorrow

I think you are missing the point, the StreetAddress is a Compound with all the attributes, you should not redefine all them in the model. See LatLong usage in the Country Model in this test project. StreetAddress could be just a property of Country, and would allow to save / Validate the full address components